src/Akeneo/Connectivity/Connection/front/src/model/Apps/monitor-settings.ts

```diff
@@ -0,0 +1,6 @@
+import {FlowType} from '../flow-type.enum';
+
+export type MonitorSettings = {
```

Comment:
```suggestion
export type MonitoringSettings = {
```
to be consistent with the back